features/OCM_disable_rule.feature

```diff
@@ -0,0 +1,109 @@
+Feature: Disabling rule in OCM user interface
+
+
+  Scenario: Check if OCM user interface is accessible for two test accounts
+    Given OCM user interface is accessible
+      And CCX external data pipeline is in working state
+      And user U1 exists for test organization
+      And user U2 exists for test organization
+     When user U1 tries to login to OCM UI
```

Comment:
What about, instead of "Then he should be logged in", specifying it with the concrete user?

"Then user U1 should be logged in"
...
"Then user U2 should be logged in"

Reply:
yes, good idea, thank you

Reply:
fixed now